.github/workflows/release.yml

```diff
@@ -0,0 +1,115 @@
+name: Release
+on: workflow_dispatch
+permissions:
+  contents: read
+
+jobs:
+  prepare:
+    permissions:
+      contents: write
+    runs-on: ubuntu-latest
+    outputs:
+      version: ${{ steps.update-version.outputs.version }}
+      head_sha: ${{ steps.push_release.outputs.head_sha }}
+
+    steps:
+      - uses: actions/checkout@v3
+        with:
+          fetch-depth: 0
+
+      - uses: actions/setup-python@v4
+        with:
+          python-version: "3.10"
+
+      - name: Update version
+        id: update-version
+        run: |
+          python devscripts/update-version.py | grep -Po "version=\d+\.\d+\.\d+(\.\d+)?" >> "$GITHUB_OUTPUT"
+          make issuetemplates
+
+      - name: Push to release
+        id: push_release
+        run: |
+          git config --global user.name github-actions
+          git config --global user.email github-actions@example.com
+          git add -u
+          git commit -m "[version] update" -m "Created by: ${{ github.event.sender.login }}" -m ":ci skip all :ci run dl"
+          git push origin --force ${{ github.event.ref }}:release
+          echo "head_sha=$(git rev-parse HEAD)" >> "$GITHUB_OUTPUT"
+
+      - name: Update master
+        if: vars.PUSH_VERSION_COMMIT != ''
+        run: git push origin ${{ github.event.ref }}
+
+  publish_pypi_homebrew:
+    needs: prepare
+    runs-on: ubuntu-latest
+
+    steps:
+      - uses: actions/checkout@v3
+      - uses: actions/setup-python@v4
+        with:
+          python-version: "3.10"
+
+      - name: Install Requirements
+        run: |
+          # TODO: Investigate required installs
```

Comment:
?

Reply:
Currently this is just the copied packages from the previous merged step. Since I don't know what exactly is required by setup.py/brew to work I left it pending for now. Somebody with knowledge of the setup.py build process should maybe investigate. Worst case we leave it like this and it installs more packages than needed.

Reply:
Feel free to commit changes and the TODO removal as wanted or let me dive in and resolve this in depth tomorrow.

Reply:
So for this job it would become
```yml
      - name: Install Requirements
        run: |
          python -m pip install -U pip setuptools wheel twine
          python -m pip install -U -r requirements.txt
```